global.json

```
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "3.1.302",
-    "rollForward": "latestPatch"
+    "version": "3.1",
```

```suggestion
    "version": "3.1.302",
```
I'd like to keep .302 as a minimum patch number. However, I'm more than happy to keep the latestFeature roll-forward. This isn't really too important for master, however 2.0 in particular often requires specific versions of the .NET SDK due to the features we use. As such, I'd like to follow suit here.